### PR TITLE
Design tweaks

### PR DESCRIPTION
**Pull request by LukasHirt**
Updated design system, adjusted all necessary icon sizes and spacing and changed the wording of "People".

### CODE REVIEW

File: apps/files/src/components/FileList.vue [hunk 1]
```diff
@@ -60,7 +60,7 @@
             >
               <div v-if="checkboxEnabled">
                 <oc-checkbox
-                  class="uk-margin-small-left"
+                  class="uk-margin-small-left files-list-checkbox"
```
Comment (kulmann) [inline apps/files/src/components/FileList.vue:67]: You are already using the size property (needs to be changed to `large` though). Don't use the `files-list-checkbox` class (it's removed already anyway).

File: apps/files/src/components/FileList.vue [hunk 2]
```diff
@@ -322,12 +330,21 @@ export default {
...
 
     $_resizeHeader() {
-      this.$nextTick(() => {
+      setTimeout(() => {
         const headerRow = this.$refs.headerRow
         const firstRow = this.$refs.firstRow
+        const headerCheckbox = document.querySelector('#files-list-header-checkbox')
+        const firstRowCheckbox = document.querySelector('#files-list-row-checkbox')
+
+        console.log(headerCheckbox)
```
Comment (kulmann) [inline apps/files/src/components/FileList.vue:339]: Remove the log line

**Review by kulmann — CHANGES_REQUESTED** [review]

**Review by kulmann — APPROVED** [review]

File: apps/files/src/components/Collaborators/SharedFilesList.vue [hunk 3]
```diff
@@ -23,7 +23,7 @@
         key="shared-with-header-cell"
         class="uk-visible@s uk-text-nowrap uk-text-meta uk-width-medium uk-text-right"
         translate-context="'People' table column"
-        v-text="$gettext('People')"
+        v-text="$gettext('Shared with')"
```
Comment (kulmann) [inline apps/files/src/components/Collaborators/SharedFilesList.vue:26]: I'm pretty sure that this just went wrong during rebasing... should be `People`

Comment (kulmann) [reply apps/files/src/components/Collaborators/SharedFilesList.vue:26]: Or was it on purpose? @LukasHirt

Comment (LukasHirt) [reply apps/files/src/components/Collaborators/SharedFilesList.vue:26]: On purpose

Comment (kulmann) [reply apps/files/src/components/Collaborators/SharedFilesList.vue:26]: 👍

**Pull request merged**